src/Opdex.Platform.WebApi/Mappers/PlatformWebApiMapperProfile.cs

```diff
@@ -59,8 +59,11 @@ public class PlatformWebApiMapperProfile : Profile
     public PlatformWebApiMapperProfile()
     {
         CreateMap<WrappedTokenDetailsDto, WrappedTokenDetailsResponseModel>()
+            .ForMember(dest => dest.Custodian, opt => opt.MapFrom(src => src.Custodian))
             .ForMember(dest => dest.Chain, opt => opt.MapFrom(src => src.Chain))
             .ForMember(dest => dest.Address, opt => opt.MapFrom(src => src.Address))
+            .ForMember(dest => dest.ModifiedBlock, opt => opt.MapFrom(src => src.ModifiedBlock))
```

Comment:
I think we might do this somewhere else too, I don't recall exactly, but it might make sense to just return CreatedBlock too everywhere so its consistent for clients, even if it won't change or matches its parent. 

Not a huge deal but maybe something to consider